.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
+extend-ignore = E203
```

Comment:
Ignoring this since black is adamant about leaving a whitespace before the colon. https://www.flake8rules.com/rules/E203.html